accessibility-checker-engine/help/WCAG20_Html_HasLang.mdx

```diff
@@ -28,21 +28,21 @@ Speech synthesizers that support multiple languages can adapt to the pronunciati
 ### What to do
 
 * Identify the default language of the HTML document by providing the `lang` attribute on the HTML element (e.g. `<html lang="en">`);
-* OR, if serving an XML document only (ie. using a MIME type such as application/xhtml+xml), identify the default language by providing the `xml:lang` attribute alone. The `lang` attribute is not needed (`<html xml:lang="fr" xmlns="https://www.w3.org/1999/xhtml">`)
-* OR, if serving XHTML 1.x or polyglot pages as text/html, use both the `lang` attribute and the `xml:lang` attribute and ensure the values for both attributes are identical (`<html lang="fr" xml:lang="fr" xmlns="https://www.w3.org/1999/xhtml">`)
+* **Or**, if serving an XML document only (ie. using a MIME type such as application/xhtml+xml), identify the default language by providing the `xml:lang` attribute alone. The `lang` attribute is not needed (`<html xml:lang="fr" xmlns="https://www.w3.org/1999/xhtml">`)
+* **Or**, if serving XHTML 1.x or polyglot pages as text/html, use both the `lang` attribute and the `xml:lang` attribute and ensure the values for both attributes are identical (`<html lang="fr" xml:lang="fr" xmlns="https://www.w3.org/1999/xhtml">`)
 
 </Column>
 <Column colLg={5} colMd={3} colSm={4} className="toolLeft">
 
 ### About this requirement
 
-[IBM 3.1.1 Language of Page](https://www.ibm.com/able/requirements/requirements/#3_1_1)
-[WCAG 2.1 technique H57](https://www.w3.org/WAI/WCAG21/Techniques/html/H57)
+* [IBM 3.1.1 Language of Page](https://www.ibm.com/able/requirements/requirements/#3_1_1)
+* [WCAG technique H57](https://www.w3.org/WAI/WCAG21/Techniques/html/H57)
 
 ### Who does this affect?
 
-* People using a screen reader, including blind, low vision and neurodivergent people
-* People with dyslexia
+ * People using a screen reader, including blind, low vision and neurodivergent people
+ * People with dyslexia
```

Comment:
Why spaces added here? Won't that indent? I don't see spaces on the other bullets

Reply:
This was an attemt to fix an mdx formatting bug.  Removing the space and indent and retesting.

Reply:
Formats correctly now